src/sagemaker/workflow/steps.py

```diff
@@ -93,6 +93,26 @@ def ref(self) -> Dict[str, str]:
         return {"Name": self.name}
 
 
+@attr.s
+class CacheConfig:
+    """Step to cache pipeline workflow.
+
+    Attributes:
+        enable_caching (bool): To enable step caching. Off by default.
+        expire_after (str): If step caching is enabled, a timeout also needs to defined.
+            It defines how old a previous execution can be to be considered for reuse.
```

Comment:
Is there a default value for this? If there is a default value for the `enable_caching`, there should be one for subsequent arguments, too.